system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -29,7 +29,7 @@ namespace system_metrics_collector
 PeriodicMeasurementNode::PeriodicMeasurementNode(
   const std::string & name,
   const std::chrono::milliseconds measurement_period,
-  const std::string & publishing_topic,
+  const std::string & publishing_topic,   // NOLINT
```

Comment:
Can't we fix this? Generally, each NOLINT **must** contain a justification of why the linting rules has been disabled.

Reply:
The issue the linter caught here was https://clang.llvm.org/extra/clang-tidy/checks/modernize-pass-by-value.html

I am not sure it warrants an API change. Any thoughts, @zmichaels11, @dabonnie, @emersonknapp?

I tried providing a justification with `// NOLINT(modernize-pass-by-value)`, which `clang-tidy` is happy about but `cpplint` is not (it complained about unrecognized `NOLINT` criteria).

Reply:
Can you add another comment explaining the `// NOLINT`

`modernize-pass-by-value` is also something we should probably look more into as a team.

Reply:
I am thinking we disregard `modernize-pass-by-value` (turn off `modernize-pass-by-value` checks) until we look more into it as a team, so `NOLINT` will not be applicable here for now.